core/src/main/java/org/apache/iceberg/BaseMetadataTable.java

```diff
@@ -192,6 +192,11 @@ public List<StatisticsFile> statisticsFiles() {
     return ImmutableList.of();
   }
 
+  @Override
```

Comment:
Doesn't need to be implemented here because we have the default "not supported" above